### PR TITLE
Update README.md to include basic install guide

### PR DESCRIPTION
**Pull request by SammyIsConfused**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,3 +15,64 @@ Support
...
 that you want to contribute back. Opening issues
 [here](https://github.com/equella/moodle-mod_equella/issues) is also recommended.
+
+Docker Installation For Testing Purposes
+----------------------------------------
+ 
+Clone [this link](https://github.com/jmhjjardison/docker-moodle) to get a docker instance of Moodle.
+Then use Docker to build it.
+
+```
```
Comment (ChristianMurphy) [inline README.md]: :art: include language flag for code block
```suggestion
```sh
```

File: README.md [hunk 2]
```diff
@@ -15,3 +15,64 @@ Support
...
+git clone https://github.com/jmhardison/docker-moodle
+cd docker-moodle
+docker build -t moodle
+```
+
+Then setup and run the MYSQL database for use with the docker Moodle.
+ 
+```
```
Comment (ChristianMurphy) [inline README.md]: :art: include language flag for code block
```suggestion
```sh
```

File: README.md [hunk 3]
```diff
@@ -15,3 +15,64 @@ Support
...
+docker run -d --name DB -p 3306:3306 -e MYSQL_DATABASE=moodle -e MYSQL_ROOT_PASSWORD=moodle -e MYSQL_USER=moodle -e MYSQL_PASSWORD=moodle mysql
+```
+
+Then, run the Moodle instance. Give it a URL  and a matching port. 
+
+```
```
Comment (ChristianMurphy) [inline README.md]: :art: include language flag for code block
```suggestion
```sh
```

File: README.md [hunk 4]
```diff
@@ -15,3 +15,64 @@ Support
...
+
+__NOTE:__ 
+This port and URL should not conflict with that of your openEQUELLA.
+
+From this point on, you should open the Moodle instance in your web browser
+and follow the installation process.
+ 
+You can access the Terminal of your Moodle if you so wish with the following command:
+
+```
```
Comment (ChristianMurphy) [inline README.md]: :art: include language flag for code block
```suggestion
```sh
```

File: README.md [hunk 5]
```diff
@@ -15,3 +15,64 @@ Support
...
+docker exec -it moodle bash
+```
+
+This project folder should be copied into the Moodle at `/var/www/html/mod`. It must be renamed from `moodle-mod_equella`
+to simply `equella` in order to work properly. From your machine terminal in the folder that contains the `moodle-mod_equella` directory:
+
+```
```
Comment (ChristianMurphy) [inline README.md]: :art: include language flag for code block
```suggestion
```sh
```

**Comment by cbeach47** [comment]
nice :+1:

**Pull request merged**